### PR TITLE
Minor form changes, use more of the Phoenix bits

### PR DESCRIPTION
**Pull request by houli**

### CODE REVIEW

**Review by c-brenn — COMMENTED** [review]
some guidance on the javascript changes would be 💯

**Review by c-brenn — APPROVED** [review]
I hereby approve eoin's changes 🗝

File: panacea/web/static/js/app.js [hunk 1]
```diff
@@ -16,30 +16,28 @@ async function submitFile() {
...
+const errorResultMessage = document.getElementById('error-result-message');
+const successPanel = document.getElementById('success-panel');
+const errorPanel = document.getElementById('error-panel');
+
 const renderFileResponse = data => {
-  const successResultMessage = document.getElementById('success-result-message');
-  const errorResultMessage = document.getElementById('error-result-message');
-  const successPanel = document.getElementById('success-panel');
-  const errorPanel = document.getElementById('error-panel');
+    successResultMessage.innerHTML = JSON.stringify(data.drugs);
```
Comment (houli) [inline panacea/web/static/js/app.js]: Scoot these back over to 2 space

File: panacea/web/static/js/app.js [hunk 2]
```diff
@@ -16,30 +16,28 @@ async function submitFile() {
...
+    errorPanel.style.display = 'none';
+    successPanel.style.display = 'block';
+};
 
-  if (data.status === 'error') {
-      // TODO: When API stops sending 200s change this
-    errorResultMessage.innerHTML = data.message
+const renderFileError = data => {
+    errorResultMessage.innerHTML = data.message;
```
Comment (houli) [inline panacea/web/static/js/app.js]: Make `errorResultMessage` and `successResultMessage` local to just the functions that need them?

**Comment by houli** [comment]
I herebleurgh the bleurgh 👍

**Comment by c-brenn** [comment]
I hope Circle herbee blurgs

**Pull request merged**